### ISSUE TITLE
Publication

### ISSUE THREAD

**Issue by chrzyki** [lexibank/chaconcolumbian#4]
@LinguList Just to verify: Does this dataset refer to the same publication as chaconarawakan, because it is concerned with the areal word list by Huber and Reed _Vocabulario Comparativo: Lista de PalabrasSelectas En Lenguas Indigenas de Colombia_? (in contrast to the Swadesh 100 for the other part). Or am I missing something?

**Comment by LinguList** [comment]
If the metadata quotes the same paper: yes. I think I received two files from T. Chacon by then, after I asked him to put the data online.

**Comment by LinguList** [comment]
In fact, the data is different, though: I coded the arawakan Data independently from a spreadsheet Tiago had prepared from sources for him in edictor format. Then, I also gave him access to Huber1992, which is where this dataset comes from. So the Arawakan sub-part was coded twice. And both datasets supplement the paper, although the paper itself does not really make clear what was used when. Something we have to live with...
